resources/splunk_install.rb

```diff
@@ -0,0 +1,219 @@
+# Encoding: utf-8
+
+class SplunkInstall < ChefCompat::Resource
+  include PlatformHelpers, PathHelpers
+
+  resource_name :splunk_install
+
+  property :name, String, name_property: true, identity: true
+  property :package, [:splunk, :universal_forwarder], required: true
+  property :service_user, String, default: 'splunk'
```

Comment:
We don't seem to be using the user at all. Windows doesn't require it, but it needs to be set at install time I think. User creation should be part of service setup (which is intertwined with the Windows installation process but we can deal with that when we get to it).

I'm probably going to remove user creation then and it will be part of the service resource.